stream-chat-android-client/src/main/java/io/getstream/chat/android/client/models/AppSettings.kt

```diff
@@ -18,11 +16,10 @@ public data class AppSettings(
  * @param fileUploadConfig [FileUploadConfig] The configuration of file uploads.
  * @param imageUploadConfig [FileUploadConfig] The configuration of image uploads.
  */
-@JsonClass(generateAdapter = true)
 public data class App(
     val name: String,
     val fileUploadConfig: FileUploadConfig,
-    val imageUploadConfig: FileUploadConfig
+    val imageUploadConfig: FileUploadConfig,
```

Comment:
The same name used for both fields in Swagger schema, so I'm betting on the fact that `ImageUploadConfig` and `FileUploadConfig` will not evolve independently.